webapp/index.html

```diff
@@ -58,14 +58,6 @@ <h1 class="display-3">Shipping Application</h1>
             </div>
             <p></p>
             <div class="row">
-                <div class="col col-lg-1">
-                    Size
-                </div>
-                <div class="col col-lg-2">
-                    <select class="custom-select custom-select-sm" id="sizeSelect">
-                        <option value="empty" selected>Size</option>
-                    </select>
-                </div>
```

Comment:
was this intended to be committed?

Reply:
It was added by an error when doing the marge, it was verified with Marin.

Reply:
Yup, Karen, please merge your personal branch into this branch to resolve this, thanks!